website/content/docs/enterprise/license/faq.mdx

```diff
@@ -56,15 +56,18 @@ The in-storage license feature will not be supported starting with Consul Enterp
 
 ## Q: What is the impact on evaluation licenses due to this change?
 
-The 6-hour trial period for evaluation licenses will be deprecated as of Consul Enterprise 1.10.0.
-This means that any clusters deployed with Consul 1.10.0+ent binaries will need to have a valid license on the disk (auto-loaded) or as an environment variable.
-Failure to provide a valid license key will result in the Consul server agent not starting.
+Consul Enterprise 1.10.0+ server agents require a valid license to start.
+The license can be provided on disk (auto-loaded) or as an environment variable.
 
 ## Q: Is there a grace period when licenses expire?
 
-A license includes an `expiration_date` and a `termination_date`. An enterprise binary will cease to function once the `termination_date` has passed.
-Licenses will now have a 24 hrs grace period. The grace period is the time between license expiry until the time it terminates.
-As Consul Enterprise approaches the expiration date, warnings will be issued in the system logs.
+**Yes—Consul will continue normal operation after license *expiration*** as defined in
+[`expiration_time`](/api-docs/operator/license#getting-the-consul-license).
+As license expiration is approached or passed, Consul will issue warnings in the system logs.
+
+Consul will only cease operation after license *termination*, which occurs 10 years
```

Comment:
10 years is a long time. So what's stopping anyone from not renewing the license and have Consul for another 10 yrs?